start-tests.sh

```diff
@@ -239,4 +388,7 @@ if [[ "$SKIP_UI_TEST" == false ]]; then
   fi
 fi
 
+# Clean up kube config
+rm -rf ./kube/config
```

Comment:
Cleaning up kubeconfig files here. After the test, we can remove the directory that was created.